conanfile.py

```diff
@@ -77,6 +77,13 @@ def package(self):
         cmake.configure(build_folder=self.build_subfolder)
         cmake.install()
         
```

Comment:
are these explicit copying instructions really necessary? I guess once you define `WINDOWS_EXPORT_ALL_SYMBOLS` DLL and LIB files are created, and `cmake.install` should copy them properly, could you please double-check?

Reply:
okay, let's leave only .dll copying then